index.js

```diff
@@ -0,0 +1,384 @@
+const GameBoard = (() => {
+  let origBoard = [''];
+  const form = document.querySelector('#form');
+  const player2InputField = document.querySelector('.user-input2');
+  const player1InputField = document.querySelector('.user-input1');
+  const displayPlayerName = document.querySelector('.player-text');
+  const cells = document.querySelectorAll('.cell');
+  const restart = document.querySelectorAll('.restart');
+  const alert = document.querySelector('.invalid');
+  const categories = document.querySelectorAll('.selection');
+  const arr = new Array(categories.length).fill(false);
+  const modes = document.querySelectorAll('.mode-text');
+  const selectedMode = { multiplayer: true, aiEasyMode: false, aiHardMode: false };
+  const player = (player, token, imgLink) => ({ player, token, imgLink });
+  const player1Name = player('', 'X', 'https://img.icons8.com/ios-filled/160/000000/x.png');
+  const player2Name = player('player', 'O', 'https://img.icons8.com/ios-filled/100/000000/o.png');
+  const aiPlayer = player('AI', 'O', 'https://img.icons8.com/ios-filled/100/000000/o.png');
+  let isAiTurnOver = true;
+  let indexOfExistingMove;
+  let playerName;
```

Comment:
Let's separate the DOM from application logic, you can create another file for DOM.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/